SRC/css-grids/css-grids.css

```diff
@@ -0,0 +1,59 @@
+.mainContainer{
+    background-image: url(https://cdn.cjr.org/wp-content/uploads/2019/07/AdobeStock_100000042-e1563305717660-686x371.jpeg);
+     display: grid;
+    grid-template-columns: 33% 33% 33%; 
+    grid-template-rows: 2fr 3fr 1fr;
+   /* grid-template-areas:  
```

Comment:
because we set the body height at 100vh (view-height), we need to set the main-container to be 100% of the available space.
add : `height: 100%;`